oio_rest/tests/test_rest_dokument.py

```diff
@@ -0,0 +1,239 @@
+import unittest
+import uuid
+
+from oio_rest.utils import test_support
+from oio_rest.utils.build_registration import is_uuid
+
+
+class TestDokument(test_support.TestRestInterface):
+    @unittest.skip("I don't know what should happen here.")
+    def test_create_dokument_empty_dict(self):
+        result = self.client.post(
+            "/dokument/dokument",
+            data={"json": "{}"},
+        ).get_json()
+        assert result["uuid"] is not None
+
+    def test_create_dokument_missing_files(self):
+        result = self.client.post(
+            "/dokument/dokument",
+            data={
+                "json": open("tests/fixtures/dokument_opret.json", "rb").read(),
+            }
+        ).get_json()
+        assert "uuid" not in result and result["message"]
```

Comment:
Likewise:

```suggestion
        self.assertNotIn("uuid", result)
        self.assertTrue(result["message"])
```

…or perhaps something even more thorough.